packages/edit-site/src/hooks/commands/use-common-commands.js

```diff
@@ -288,6 +310,91 @@ function useGlobalStylesOpenRevisionsCommands() {
 		isEditorPage,
 		getCanvasMode,
 		setCanvasMode,
+		closeGeneralSidebar,
+	] );
+
+	return {
+		isLoading: false,
+		commands,
+	};
+}
+
+function useGlobalStylesStyleBookCommands() {
+	const {
+		openGeneralSidebar,
+		closeGeneralSidebar,
+		setEditorCanvasContainerView,
+		setCanvasMode,
+	} = unlock( useDispatch( editSiteStore ) );
+	const { getCanvasMode, getEditorCanvasContainerView } = unlock(
+		useSelect( editSiteStore )
+	);
+	const canvasContainerView = getEditorCanvasContainerView();
+	const { params } = useLocation();
+	const isMobileViewport = useViewportMatch( 'medium', '<' );
+	const isEditorPage = ! getIsListPage( params, isMobileViewport );
+	const history = useHistory();
+	const { isStyleBookOpened, isRevisionsStyleBookOpened, isRevisionsOpened } =
+		useSelect( () => {
+			return {
+				isStyleBookOpened: 'style-book' === canvasContainerView,
+				isRevisionsStyleBookOpened:
```

Comment:
I'm thinking it might make the open/close logic more straight forward for users to have to choice to toggle revisions regardless of whether the style book view is open. 

So something like:
```
isStyleBookOpened: 'style-book' === canvasContainerView,
isRevisionsOpened: canvasContainerView.startsWith( 'global-styles-revisions' ),
```

So "Open/Close Revisions" ignores which type of preview we're showing - the commands will either open and/or close revisions. 

"Open/Close style book" would target the style book specifically. 

I mention it because the command "Close style book" is present in the context of revisions:

<img width="1104" alt="Screenshot 2024-01-29 at 11 13 58 am" src="https://github.com/WordPress/gutenberg/assets/6458278/5c37dbad-b59b-49bd-9bb5-8d2700566c1f">

I suppose it's okay to have a command that closes the style book revisions "view" too - I just have a niggling feeling the third state is unnecessary for the user.